source/extensions/all_extensions.bzl

```diff
@@ -6,7 +6,6 @@ load("@envoy_build_config//:extensions_build_config.bzl", "EXTENSIONS")
 _required_extensions = {
     "envoy.http.original_ip_detection.xff": "//source/extensions/http/original_ip_detection/xff:config",
     "envoy.request_id.uuid": "//source/extensions/request_id/uuid:config",
```

Comment:
removing here is balanced out by adding to the exe explicitly.
I think it's more consistent to do that for required extensions, especially as we believe the openssl buid is stale